src/Chart/Draw.php

```diff
@@ -8051,8 +8051,8 @@ public function drawStackedBarChart(array $Format = [])
 
         $RestoreShadow = $this->Shadow;
 
-        $LastX = "";
-        $LastY = "";
+        $LastX = null;
```

Comment:
Hey, at a first glance I think it would make more sense to initialize these variables as arrays, don't you think?